src/scripts/bash_functions.sh

```diff
@@ -355,29 +355,27 @@ load_web_password_secret() {
 }
 
 setup_web_password() {
-    if [ -z "${WEBPASSWORD+x}" ] ; then
+    if [ -z "${WEBPASSWORD_OVERRIDE+x}" ] ; then
         # ENV WEBPASSWORD is not set
 
         # Exit if setupvars already has a password
         setup_var_exists "WEBPASSWORD" && return
-
         # Generate new random password
         WEBPASSWORD=$(tr -dc _A-Z-a-z-0-9 < /dev/urandom | head -c 8)
         echo "Assigning random password: $WEBPASSWORD"
     else
         # ENV WEBPASSWORD is set an will be used
```

Comment:
Since you are fixing it, the comment is wrong.

```diff
- # ENV WEBPASSWORD is set an will be used
+ # ENV WEBPASSWORD is set and will be used
```